pyquat/plot.py

```diff
@@ -70,21 +72,21 @@ def plot_frame(q, r = numpy.zeros((3,1)), fig = None, axes = None, axis_size = 1
     if axes is None:
         axes = fig.add_subplot(111, projection='3d')
 
-    xh = numpy.zeros((3,2))
+    xh = numpy.zeros((3, 2))
     yh = numpy.zeros_like(xh)
     zh = numpy.zeros_like(xh)
-    xh[0,1] = axis_size
-    yh[1,1] = axis_size
-    zh[2,1] = axis_size
+    xh[0, 1] = axis_size
+    yh[1, 1] = axis_size
+    zh[2, 1] = axis_size
```

Comment:
Okay, I'm going to push back on the second part of this one a bit more because it doesn't actually appear in pep8: https://stackoverflow.com/questions/31741418/pep8-space-after-a-comma/37064013#37064013

I think there should be spaces after commas *except* within array notation. The examples in pep8 do show spaces after commas, even though this isn't specifically enumerated; but they don't show multi-dimensional array indexing notation.

Thoughts?